spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigMapPropertySource.java

```diff
@@ -44,30 +44,32 @@
 	private static final Log LOG = LogFactory.getLog(Fabric8ConfigMapPropertySource.class);
 
 	public Fabric8ConfigMapPropertySource(KubernetesClient client, String name) {
-		this(client, name, null, null, "");
+		this(client, name, null, null, "", true);
 	}
 
-	public Fabric8ConfigMapPropertySource(KubernetesClient client, String applicationName, String namespace,
+	@Deprecated
+	public Fabric8ConfigMapPropertySource(KubernetesClient client, String name, String namespace,
 			Environment environment) {
-		super(getName(applicationName, getApplicationNamespace(client, namespace)),
-				getData(client, applicationName, getApplicationNamespace(client, namespace), environment, ""));
```

Comment:
I've replaced `applicationName` with `name`, meaning "config map name". The config map name can be defaulted to application name, but not always - so this variable name is very miss-leading here. A simple change brings a lot more clarity